cs_scanimage.py

```diff
@@ -264,9 +264,8 @@ def __call__(self, r):
 # See https://stackoverflow.com/questions/10551117/setting-options-from-environment-variables-when-using-argparse/10551190#10551190
 class EnvDefault(argparse.Action):
     def __init__(self, envvar, required=True, default=None, **kwargs):
-        if not default and envvar:
```

Comment:
@isimluk I don't see a downside in removing this logic but might be blind to the consequences of removing it.

Reply:
seems to be fine. Let's merge and learn.

Reply:
good attitude!